homeassistant/components/sabnzbd/quality_scale.yaml

```
@@ -0,0 +1,81 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: exempt
+    comment: |
+      The integration has deprecated the actions, thus the documentation has been removed.
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: todo
+  entity-event-setup:
+    status: exempt
+    comment: |
+      Entities of this integration does not explicitly subscribe to events.
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
```

```suggestion
  runtime-data: todo
```
untyped entry in coordinator

---

Nice, thanks for spotting 👍

---

Fixed with https://github.com/home-assistant/core/pull/132098